Alignment/OfflineValidation/test/BuildFile.xml

```diff
@@ -1,33 +1,34 @@
 <environment>
-  <bin file="testAlignmentOfflineValidation.cpp" name="testAlignmentOfflineValidation">
-    <flags TEST_RUNNER_ARGS=" /bin/bash	Alignment/OfflineValidation/test test_all-in-one.sh"/>
+  <bin file="testAlignmentOfflineValidation.cpp" name="validateAlignments">
+    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test validateAlignments.sh"/>
     <use name="FWCore/Utilities"/>
   </bin>
-  <bin file="testAlignmentOfflineValidation.cpp" name="testPVValidation">
-    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test testPVValidation.sh"/>
+  <bin file="testAlignmentOfflineValidation.cpp" name="DMRall">
+    <flags PRE_TEST="validateAlignments"/>
+    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test test_unitDMR.sh"/>
     <use name="FWCore/Utilities"/>
   </bin>
-  <bin file="testAlignmentOfflineValidation.cpp" name="testMiscellanea">
-    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test testMiscellanea.sh"/>
+  <bin file="testAlignmentOfflineValidation.cpp" name="PVall">
+    <flags PRE_TEST="validateAlignments"/>
+    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test test_unitPV.sh"/>
     <use name="FWCore/Utilities"/>
   </bin>
-  <bin file="testAlignmentOfflineValidation.cpp" name="testEoP">
-    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test testEoP.sh"/>
+  <bin file="testAlignmentOfflineValidation.cpp" name="PVrbr">
+    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test test_unitPVrbr.sh"/>
     <use name="FWCore/Utilities"/>
   </bin>
-  <bin file="testAlignmentOfflineValidation.cpp" name="testSubmitters">
-    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test testSubmitters.sh"/>
```

Comment:
please keep this test.

Reply:
already kept elsewhere

Reply:
please keep it separate.

Reply:
what's the reason?

Reply:
And here you mean to keep submitters from the rest in test_unitPVrbr.sh? Or also to keep splitV submitter from PV submitter. Be specific please

Reply:
So that it's easy to disentangle failures in PR and IB tests

Reply:
Given there is specified error message per each python/C++ macro run I see no difference besides possible parallelization in case that there is just one macro per each test file. But okay, I will split it as much as possible.